code/game/objects/items/devices/flashlight.dm

```diff
@@ -202,6 +202,8 @@
 	on = FALSE
 	force = initial(force)
 	damtype = initial(damtype)
+	hitsound = "swing_hit"
+	attack_verb = list()
```

Comment:
It wont have this sound/verb if the player didnt turn on/off at least once right? It should be moved to the item itself then making it return to its initial() sound/verbs here.

Reply:
Looking at the way its being set over /item i guess this is fine actually, you can resolve this review